### ISSUE TITLE
[Feature] Filter feature for packet log interface

### ISSUE THREAD

**Issue by ghost** [henrypp/simplewall#1837]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same feature request.
- [X] I will describe the problem with as much detail as possible.
- [X] This issue only contains a request for one single feature, **not** multiple (related) features.

### App version

3.8.2

### Problem you are trying to solve

When enabling the packet logging interface feature and allowing allowed connections to be logged, some apps cause simplewall to freeze due to how many requests they make in a short amount of time. Similar to [1807](https://github.com/henrypp/simplewall/issues/1807) and [1684](https://github.com/henrypp/simplewall/issues/1684)

### Suggested solution

A filtering feature for the packet log interface to exclude specifics apps from having their packets logged would be very nice. thank you :blush:

**Comment by garandal245** [comment]
yes this would help so much when trying to narrow down whats causing a specific program to be blocked when it shouldnt, as currently packet log is pretty useless such as wrmgr due to how much it spams it

**Comment by V29G** [comment]
I'm encountering exactly this issue in simplewall 3.9.1 with gamingservices.exe.

Gaming Services repeatedly generates several blocked connections approximately every 15 seconds. In my case, it can almost completely fill the visible packet log within about 3 minutes.

I want Gaming Services to remain blocked, but I also want to keep the packet log enabled so I can monitor blocked traffic from other applications. Disabling the entire packet log works around the problem, but defeats its purpose.

A per-application “Exclude from packet log” option would solve this perfectly.

I also noticed that the simplewall changelog for v1.3.1 (2016) says:
added option to exclude application from log

So apparently this functionality existed in an older version. Would it be possible to restore it?
